src/main.ts

```diff
@@ -99,4 +113,35 @@ function renderSeverity(
   return `${styles.color[color].open}(${severity} severity)${styles.color[color].close}`
 }
 
+function printLicensesError(
+  changes: Array<Change>,
+  licenses: {
+    allow?: Array<string>
+    deny?: Array<string>
+  }
+): void {
+  if (changes.length == 0) {
+    return
+  }
+
+  let {allow = [], deny = []} = licenses
+
+  core.info('Dependency review detected incompatible licenses.')
+
+  if (allow.length > 0) {
+    core.info('\nAllowed licenses: ' + allow.join(', ') + '\n')
```

Comment:
Optional: I wonder if it might be least confusing to just display the ineligible licenses the scan found?

Reply:
I think good to show the users what's in their config files in case there's a failure, saves you a trip to the YAML file. If this causes any type of confusion I'll be happy to change this behavior!